lib/view/bbcode/CsrBB.class.php

```diff
@@ -144,15 +154,63 @@ public static function filterCommentaar($bbcode) {
 	}
 
 	/**
-	 * Dit laad de twitter account van het hidden cash spel.
+	 * Templates for light mode
 	 */
-	function bb_hidden($arguments = array()) {
-		$html = '<a class="twitter-timeline" href="https://twitter.com/HiddenCashCSR" data-widget-id="477465734352621568">Tweets by @HiddenCashCSR</a>
-							<script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http://.test(d.location)?\'http\':\'https\';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+"://platform.twitter.com/widgets.js";fjs.parentNode.insertBefore(js,fjs);}}(document,"script","twitter-wjs");</script>';
-		return $html;
+	public static function lightLinkInline($tag, $url, $content) {
```

Comment:
(eventueel voor een volgende refactor)
Beter deze functie niet static, wat er in `getBijbelLink` gebeurt is niet heel netjes, het lijkt me beter als dat de uitzondering is. `static` functies aanroepen in instance context is ook niet echt netjes.

Je kan beter in `getBijbelLink` het volgende doen:
```
return (new CsrBB())->lightLinkInline('bijbel', $link, $stukje);
```